app/assets/javascripts/lookup_keys.js

```diff
@@ -69,7 +69,7 @@ function add_child_node(item) {
     var field   = '';
     if (assoc == 'lookup_keys') {
       $('#smart_vars .smart-var-tabs .active, #smart_vars .stacked-content .active').removeClass('active');
-      var pill = "<li class='active'><a data-toggle='pill' href='#new_" + new_id + "' id='pill_new_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
+      var pill = "<li class='active'><a data-toggle='pill' href='#" + new_id + "' id='pill_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
```

Comment:
maybe all of the strong params needs to be merged separately from thsi branch? @shlomizadok @eLobato ?